### DISCUSSION TITLE
Theme di R markdown

### DISCUSSION THREAD

**Discussion by 19807311** [teamalgoritma/community#450] [Q&A] [answered]
![Uploading 20221112_122421.jpg…]()

Halo, saya mau tanya saya ingin menggunakan theme `material` dari package rmdformats. Ketika sudah di knit kok theme nya tidak muncul ya, jadi tampilannya html biasa.
Bagaimana cara set theme yang benar? Terima kasih

![20221112_122432](https://user-images.githubusercontent.com/114885734/201458870-49d8ceba-9d89-498d-9c9f-de457219e6cb.jpg)
![20221112_122712](https://user-images.githubusercontent.com/114885734/201458885-9ba6c539-ba51-4226-8570-356adc3f0bcc.jpg)

**Comment by takdirzd** [comment, marked as answer]
Dear @19807311 Ibu Bidhari

Tema yang tampil di hasil knit Ibu, masih hasil dari format "html_document". Dapat dilihat bahwa pada "output :" terdapat 2 jenis output, yaitu "html_document" dan "rmdformats::material". Output hanya bisa diisi 1 jenis output. Jika Ibu ingin menggunakan tema dari package "rmdformats", maka Ibu hanya perlu menuliskan format "rmdformats" nya saja. Seperti yang tertera pada dokumentasi berikut https://bookdown.org/yihui/rmarkdown/rmdformats.html

**Reply by 19807311** [reply]
Terima kasih mas

**Reply by takdirzd** [reply]
Sama-sama Ibu Bidhari, semoga bermanfaat :)
